parse.go

```diff
@@ -360,7 +363,7 @@ func cmdFromStruct(name string, dest path, t reflect.Type) (*command, error) {
 			case strings.HasPrefix(key, "--"):
 				spec.long = key[2:]
 			case strings.HasPrefix(key, "-"):
-				if len(key) != 2 {
+				if len(key) > 2 {
```

Comment:
arguments don't get a short option by default, so users can set "no short option" just by not including any short option at all in the argument tag -- which means I think you can drop the change here